### ISSUE TITLE
alternate card/image improvements

### ISSUE THREAD

**Issue by brine** [octgn/OCTGN#779]
as I talked about in chat... the proxy gen doesn't handle very well with alternate image cards.  The only thing that really flags OCTGN to the existence of an alternate image is the physical cardname.b.jpg filename for the card image -- there's nothing in the set XMLs that hint that the card may have an alternate image.

So this is sorta an upgrade to the current alternateImage functionality, and a revamp of the SwitchWithAlternate stuff. 

`<card id="20435-34m5435-345m43whatever56" name="Card 1" name2="Card 2" >
  <property name="Type" value="Dude" value2="Other Dude" />
</card>`

so basically, we've added parallel properties (in my example, I just added a 2 to signify a secondary set of card text; final syntax subject to change) onto a card. This would signal to OCTGN that this card has multiple images, so that the proxy generator can differentiate between the two and generate the correct 'side'.

Also, by splitting each 'side' of a card into multiple elements, the deck editor and gameplay scripting can also properly differentiate each side of a card.
